graylog2-web-interface/src/components/search/MessageFieldSearchActions.jsx

```diff
@@ -23,6 +23,18 @@ const MessageFieldSearchActions = React.createClass({
     );
   },
   render() {
+    let messageField = this.props.message.fields[this.props.fieldName];
```

Comment:
Small nitpick: can we use `const` instead of `let` in here? The variable is never modified afterwards.